tests/test_integration/test_pipeline.py

```diff
@@ -114,8 +98,8 @@ def test_linear():
         "rel_tol": 1e-6,
         "max_num_steps": int(1e9),
         "likelihood": 1,
-        "n_samples": 200,
-        "n_warmup": 200,
+        "n_samples": 20,
+        "n_warmup": 20,
```

Comment:
I appreciate the idea here but maybe something like 40 would be safer?

Reply:
Oh Sorry! this was supposed to be deleted. I will change it to 40 just for testing sake